### DISCUSSION TITLE
parse and access attributes

### DISCUSSION THREAD

**Discussion by jkristia** [NaturalIntelligence/fast-xml-parser#570] [Q&A]
Is there a flag to enabled in order to have the attributes being returned in the obj ?
In this case I would like to have a `@diagram:'true'` and `@name='interface'` on the returned obj
```
<element diagram='true' name='interface'>
   ...
```

**Comment by amitguptagwl** [comment]
I'm not sure what exactly you want. But I'll suggest you to go through the documentation once.

**Comment by ChrisMBarr** [comment]
I'd recommend you take a look at the docs, but these options in particular:
* [`allowBooleanAttributes`](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/2.XMLparseOptions.md#allowbooleanattributes)
* [`parseAttributeValue`](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/2.XMLparseOptions.md#parseattributevalue)
* [`ignoreAttributes`](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/2.XMLparseOptions.md#ignoreattributes)

**Comment by mitya33** [comment]
For me, attributes are not present in the resultant JS regardless of the options I use. I'm using:
```
{
    ignoreAttributes: false,
    parseAttributeValue: true,
    allowBooleanAttributes: true
}
````

**Reply by amitguptagwl** [reply]
can you please raise an issue with sample code you're using.
